cmd/kubelet/app/server.go

```diff
@@ -207,6 +208,14 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				klog.Fatal(err)
 			}
 
+			if kubeletConfig.KubeletCgroups == "" && kubeletConfig.KubeReservedCgroup != "" {
+				klog.Warning("invalid configuration: KubeReservedCgroup (--kube-reserved-cgroups) was specified," +
+					"but KubeletCgroups (--kubelet-cgroups) was not specified,so set it equal to KubeReservedCgroup")
+				kubeletConfig.KubeletCgroups = kubeletConfig.KubeReservedCgroup
+			} else if 0 != strings.Index(kubeletConfig.KubeletCgroups, kubeletConfig.KubeReservedCgroup) {
+				klog.Warning("invalid configuration:KubeletCgroups is not within KubeReservedCgroup")
```

Comment:
maybe use "unsupported" instead of "invalid".  Invalid configuration should be fatal.  We warn when it is not recommended/supported.

Reply:
it has been modified